### PR TITLE
Add slack integration for job board

### PR DESCRIPTION
**Pull request by elmq0022**
This PR adds a custom Django command which can be scheduled to post the list of current approved jobs to slack.  Given we use Heroku I suggest we use their scheduler to post to the jobs channel daily in the early A.M.  Maybe 7AM?

### CODE REVIEW

File: chipy_org/apps/job_board/forms.py [hunk 1]
```diff
@@ -29,7 +29,9 @@ class Meta:
         ]
 
         help_texts = {
-            "affiliation": ("Is this posting affiliated with a 3rd party? Please select:"),  # pylint: disable=line-too-long
+            "affiliation": (
+                "Is this posting affiliated with a 3rd party? Please select:"
+            ),  # pylint: disable=line-too-long
```
Comment (raymondberg) [inline chipy_org/apps/job_board/forms.py:34]: Do you still need the line-too-long?

File: chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import datetime
```
Comment (raymondberg) [inline chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py:3]: (non-blocking) Import sorts are always <3

File: chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            help="""
+            The desired weekdays to post represented an integer where Monday=0, Tuesday=1, ... , Saturday=5, and Sunday=6.
+            Posts to the slack channel will only occur on the selected weekdays.
+            """,
+        )
+
+    def handle(self, *args, **options):
+        posts = JobPost.approved_and_active.all()
+
+        if posts.count() and datetime.date.today().weekday() in options["weekdays"]:
```
Comment (raymondberg) [inline chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py:38]: Truly fine in the context of adding the code, but from a style perspective this is a little muddied with the concerns of the command. Ideally, I think we'd be well-served to put most of this into a Slack facade that handles renders and requests calls.

File: chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            slack_data = {"text": msg}
+
+            response = requests.post(
+                webhook_url,
+                data=json.dumps(slack_data),
+                headers={"Content-Type": "application/json"},
+            )
+
+            if response.status_code != 200:
+                logger.info("Failed to post to slack job channel.")
```
Comment (raymondberg) [inline chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py:54]: This should probably be a raised CommandError instead-of/as-well-as a log-entry: https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/#command-exceptions

File: chipy_org/settings.py [hunk 5]
```diff
@@ -123,7 +127,7 @@ def env_list(key, defaults=None, delimiter=","):
 
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = env_var("SECRET_KEY")
-
+DAY_TO_POST = int(env_var("DAY_TO_POST", 6))
```
Comment (raymondberg) [inline chipy_org/settings.py:130]: Hmmm, what is this for? I'd push to make any settings like this human-readable. DoW math is notoriously wrong with either Sun=0 or Sun=1.

**Review by raymondberg — APPROVED** [review]
Cool to see this work!

File: chipy_org/settings.py (hunk 5 above)
Comment (elmq0022) [reply chipy_org/settings.py:130]: Leftover code will delete this.  Was going to do it as a env var, but moved it to a command argument.

File: chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py (hunk 3 above)
Comment (elmq0022) [reply chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py:38]: That's a reasonable idea and an opportunity for me to practice a design pattern.  Though I think I'll leave it until we add slack use case number two if that's cool.

File: chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py (hunk 2 above)
Comment (elmq0022) [reply chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py:3]: :hamburger:

File: chipy_org/apps/job_board/forms.py (hunk 1 above)
Comment (elmq0022) [reply chipy_org/apps/job_board/forms.py:34]: nope

**Pull request merged**